xml/System/String.xml

```diff
@@ -10567,7 +10580,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
   
  Because this method returns the modified string, you can chain together successive calls to the <xref:System.String.Replace%2A> method to perform multiple replacements on the original string. Method calls are executed from left to right. The following example provides an illustration.  
   
- [!code-csharp[System.String.Replace#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.string.replace/cs/replace2.cs#2)]
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
+
+ [!code-csharp-interactive[System.String.Replace#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.string.replace/cs/replace2.cs#2)]
```

Comment:
this sample still contains the full program. needs to be converted.

Reply:
Done in https://github.com/dotnet/samples/pull/1171